en/docs/Learn/APISecurity/Authorization/role-based-access-control-using-xacml.md

```diff
@@ -33,7 +33,7 @@ Based on the requirement, a single API is exposed to add or retrieve order info
         In an actual deployment, both these servers can [share the user store](../../../Learn/Extensions/SAML2SSO/configuring-identity-server-as-idp-for-sso.md#sharing-the-user-store) of your organization.
 
 
-2.  Share the registry of both WSO2 API Manager and WSO2 Identity Server. Refer [Sharing the registry space](https://docs.wso2.com/display/AM2xx/Configuring+External+IDP+through+Identity+Server+for+SSO#ConfiguringExternalIDPthroughIdentityServerforSSO-Sharingtheregistryspace) for the steps.
+2.  Share the registry of both WSO2 API Manager and WSO2 Identity Server. Refer [Sharing the registry space](https://docs.wso2.com/display/AM260/Configuring+External+IDP+through+Identity+Server+for+SSO#ConfiguringExternalIDPthroughIdentityServerforSSO-Sharingtheregistryspace) for the steps.
```

Comment:
Why are we pointing to AM260 docs in the AM3.0.0 doc space

Reply:
Still, we do not have the documentation for "Share the registry space".

Reply:
@bhathiya , pls confirm

Reply:
that's not correct because 2.6 configs are not working for 3.0 
let's ask the doc owner to fix it.

Reply:
Fixed with 62cd606